ste/sender-blobSymlinks_linux.go

```diff
@@ -0,0 +1,31 @@
+package ste
+
+import (
+	"errors"
+	"fmt"
+	"github.com/Azure/azure-pipeline-go/pipeline"
+	"github.com/Azure/azure-storage-azcopy/v10/common"
+)
+
+func (s *blobSymlinkSender) getExtraProperties() error {
```

Comment:
Have we tried both symlink follow and preserve options in both windows and linux to validate our change is not OS specific at any level.

Reply:
Preserve symlinks should probably be explicitly limited to Linux-- Symlinks require special permissions on Windows to create, and this is XDM-specific work.